sycl/include/sycl/detail/host_task_impl.hpp

```diff
@@ -89,5 +90,23 @@ handler::host_task_impl(FuncT &&Func) {
   setType(detail::CG::CodeplayHostTask);
 }
 
+/// Enqueues a command to the SYCL runtime to invoke \p Func once.
+template <typename FuncT>
+std::enable_if_t<detail::check_fn_signature<std::remove_reference_t<FuncT>,
+                                            void(interop_handle)>::value>
+handler::sycl_ext_oneapi_enqueue_custom_operation_impl(FuncT &&Func) {
+  throwIfActionIsCreated();
+
+  MNDRDesc.set(range<1>(1));
+  // Need to copy these rather than move so that we can check associated
+  // accessors during finalize
+  MArgs = MAssociatedAccesors;
+
+  // TODO: Should I use a new CG type here?
```

Comment:
Who is the intended reader of this comment?

Reply:
Oops good catch, thanks